bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BookKeeper.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.bookkeeper.client.api;
+
+import org.apache.bookkeeper.client.BKException;
+
+/**
+ * This is the entry point for BookKeeper client side API
+ */
+public interface BookKeeper extends AutoCloseable {
+
+    /**
+     * Start the creation of a new ledger
+     *
+     * @return a builder for the new ledger
+     */
+    CreateBuilder createLedger();
```

Comment:
There is already a createLedger, openLedger, and deleteLedger with completely different semantics. And here you aren't actually creating the ledger when you call the method, only when you execute the returned builder, so it's misleading in a way. I would suggest renaming these to createLedgerOp, openLedgerOp and deleteLedgerOp.

Reply:
@ivankelly make sense to me, I will change the name. 
Maybe something like newLedgerBuilder() instead of createLedgerOp() ?
so it will be openLedgerBuilder() and deleteLedgerBuilder()

The only pity is that these names were already discussed.
 /cc @sijie